walking:
  engine:
    double_support_ratio: 0.122898149665364
    first_step_swing_factor: 1.25516771219043
    foot_distance: 0.180850926768754
    foot_rise: 0.0721563056173029
    freq: 2.5007567279965
    trunk_height: 0.381689943860774
    trunk_phase: -0.206780966478786
    trunk_pitch: 0.0748425542905304
    trunk_pitch_p_coef_forward: -0.0955888160107112
    trunk_pitch_p_coef_turn: -1.42966197237633
    trunk_swing: 0.0651670152575642
    trunk_x_offset: -0.021930323512739
    trunk_y_offset: -0.00922574213320393
    trunk_z_movement: 0.00752048271219194
    first_step_trunk_phase: -0.5
    foot_apex_phase: 0.5
    foot_overshoot_phase: 1
    foot_overshoot_ratio: 0.0
    foot_put_down_phase: 1
    foot_z_pause: 0
    trunk_pause: 0

    kick_length: 0.09
    kick_vel: 0.2
    kick_phase: 0.75

  node:
    # update frequency of the engine
    engine_freq: 33.0

    # parameters for bioIK
    ik_timeout: 0.01

    debug_active: True

    trunkYOnlyInDoubleSupport: False

    max_step_x: 0.1
    max_step_y: 0.1
    max_step_xy: 0.1
    max_step_z: 0.05
    max_step_angular: 1.0

    vel: -1
    acc: -1
    pwm: -1

    imu_active: False
    imu_pitch_threshold: 0.19
    imu_roll_threshold: 0.4
    imu_pitch_vel_threshold: 1.3
    imu_roll_vel_threshold: 5.7
    pause_duration: 3.0

    phase_reset_active: False
    ground_min_pressure: 1.5
    phase_reset_phase: 0.25
    joint_min_effort: 35
    effort_phase_reset_active: False


  pid_trunk_pitch:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_trunk_roll:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_trunk_fused_pitch:
    p: 0.0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_trunk_fused_roll:
    p: 0.0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_trunk_gyro_roll:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_trunk_gyro_pitch:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_cop_x:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_cop_y:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_ankle_left_pitch:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_ankle_right_pitch:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_ankle_left_roll:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_ankle_right_roll:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_hip_pitch:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_hip_roll:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_foot_pos_x:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True

  pid_foot_pos_y:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: True